README.md

```diff
@@ -202,25 +194,75 @@ Veneur also honors the same "magic" tags that the dogstatsd daemon includes in t
-* `read_buffer_size_bytes` - The size of the receive buffer for the UDP socket. Defaults to 2MB, as having a lot of buffer prevents packet drops during flush!
+
+### Limits
+* `metric_max_length` - How big a buffer to allocate for incoming metric lengths. Metrics longer than this will get truncated!
+* `trace_max_length_bytes` - How big a buffer to allocate for incoming traces
+* `ssf_buffer_size` - The number of SSF packets that can be processed per flush interval
+* `read_buffer_size_bytes` - The size of the buffer we'll use to buffer socket reads. Tune this if you think Veneur needs more room to keep up with all packets.
+* `flush_max_per_body` - how many metrics to include in each JSON body POSTed to Datadog. Veneur will POST multiple bodies in parallel if it goes over this limit. A value around 5k-10k is recommended; in practice we've seen Datadog reject bodies over about 195k.
+
+### Diagnostics
+* `debug` - Should we output lots of debug info? :)
 * `sentry_dsn` A [DSN](https://docs.sentry.io/hosted/quickstart/#configure-the-dsn) for [Sentry](https://sentry.io/), where errors will be sent when they happen.
-* `stats_address` - The address to send internally generated metrics. Probably `127.0.0.1:8125`. In practice this means you'll be sending metrics to yourself. This is expected!
-* `tags` - Tags to add to every metric that is sent to Veneur. Expects an array of strings!
+* `enable_profiling` - Enables Go profiling
+
+### Sinks
+#### Datadog
+* `datadog_api_key` - Your Datadog API key
+* `datadog_api_hostname` - The Datadog API URL to post to. Probably `https://app.datadoghq.com`.
+* `datadog_trace_api_address` - The hostname to send Datadog traces to
+
+#### InfluxDB
+* `influx_address` - The hostname to send Influx metrics to
+* `influx_consistency` - The consistency level for writes
+* `influx_db_name` - The database name to write metrics to
+
+#### LightStep
+* `trace_lightstep_access_token` - The access token for sending to LightStep
+* `trace_lightstep_collector_host` - The hostname to send trace data to
+* `trace_lightstep_reconnect_period` - How often to reconnect to LightStep collectors
+
+### Plugins
+#### S3
+* `aws_access_key_id` - The AWS access key ID, used in conjunction with `aws_secret_access_key` to authenticate to AWS
+* `aws_secret_access_key` - The AWS secret access key, used in conjunction with `aws_access_key_id` to authenticate to AWS
+* `aws_region` - The region to write to
+* `aws_s3_bucket` - The bucket to write to
+
+#### Local file
+* `flush_file` - The local file path to write metrics to
+
+## Configuration via Environment Variables
+
+Veneur and venuer-proxy each allow configuration via environment variables using [envconfig](https://github.com/kelseyhightower/envconfig). Options provided via environment variables take precedent over those in config. This allows stuff like:
```

Comment:
`veneur-proxy` :)

Reply:
Hah, I actually spotted that typo before but I missed it when I copied it over :)